cmd/webhook/main.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"log"
+	"context"
+	"github.com/knative/pkg/configmap"
+	"github.com/knative/pkg/logging"
+	"github.com/knative/pkg/logging/logkey"
+	"github.com/knative/pkg/signals"
+	"github.com/knative/pkg/webhook"
+	"github.com/tektoncd/pipeline/pkg/system"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+// WebhookLogKey is the name of the logger for the webhook cmd
+const (
+	WebhookLogKey = "webhook"
+	// ConfigName is the name of the ConfigMap that the logging config will be stored in
+	ConfigName = "config-logging-triggers"
+)
+
+func main() {
+	flag.Parse()
+	cm, err := configmap.Load("/etc/config-logging")
+	if err != nil {
+		log.Fatalf("Error loading logging configuration: %v", err)
+	}
+	config, err := logging.NewConfigFromMap(cm)
+	if err != nil {
+		log.Fatalf("Error parsing logging configuration: %v", err)
+	}
+	logger, atomicLevel := logging.NewLoggerFromConfig(config, WebhookLogKey)
+
+        defer func() {
```

Comment:
nit: It looks like there are spaces here instead of tabs. @bobcatfish is there automation that should catch something like this? 😄 

```suggestion
	defer func() {
```

Reply:
Good question! I'm not sure but I would certainly hope so :S Whether there is supposed to be or not it seems like we're not catching this 🤔  seems like https://github.com/tektoncd/triggers/issues/4 was supposed to add the same linting that our other repos have :S 

Opened https://github.com/tektoncd/triggers/issues/28 !

Reply:
Thank you! I wonder if it's covered in `pipeline` 🤔😄